src/components/lab/Lab.js

```diff
@@ -71,6 +72,9 @@ class Lab extends Component {
               height={stageHeight}
             >
               <Provider store={store}>
+                <Layer>
```

Comment:
Any specific reason why `<Thermometer>` is put in its own `<Layer>`? (Rather than putting both `Thermometer` and `Lattice` in one `Layer`?)